FinSearchUnified/Helper/StaticHelper.php

```diff
@@ -823,4 +823,30 @@ public static function setPromotion(SimpleXMLElement $xmlResponse)
             ]);
         }
     }
+
+    /**
+     * @param \SimpleXMLElement $xmlResponse
+     */
+    public static function setSmartDidYouMean(SimpleXMLElement $xmlResponse)
+    {
+        $query = $xmlResponse->query;
+        $originalQuery = (string)$query->originalQuery;
+        $didYouMeanQuery = (string)$query->didYouMeanQuery;
+        $queryString = $query->queryString;
+        $queryStringType = (string)$queryString->attributes()->type;
+
+        if (((isset($originalQuery) && $originalQuery !== '') || (isset($didYouMeanQuery) && $didYouMeanQuery !== ''))
+            && $queryStringType !== 'forced') {
+            /** @var \Enlight_View_Default $view */
+            $view = Shopware()->Front()->Plugins()->ViewRenderer()->Action()->View();
+            $type = isset($didYouMeanQuery) && !empty($didYouMeanQuery) ? 'did-you-mean' : $queryStringType;
```

Comment:
The call for `isset` shouldn't be necessary as `empty` already covers what you are trying to check.